automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -0,0 +1,319 @@
+        sys.exit(1)
+    to_version = os.environ.get('TO_VERSION')
+    if to_version not in ['6.1', '6.2']:
+        print('Wrong Capsule Version Provided to upgrade to. '
+              'Provide one of 6.1, 6.2')
+        sys.exit(1)
+    if admin_password is None:
+        admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
+    # Setting Capsule61 Repos
+    major_ver = distro_info()[1]
+    if os.environ.get('CAPSULE_URL') is None:
+        enable_repos('rhel-{0}-server-satellite-capsule-{1}-rpms'.format(
+            major_ver, to_version))
+    disable_repos('rhel-{0}-server-satellite-capsule-{1}-rpms'.format(
+        major_ver, from_version))
+    # Stop katello services, except mongod
+    run('for i in qpidd pulp_workers pulp_celerybeat '
+        'pulp_resource_manager httpd; do service $i stop; done')
+    run('yum clean all', warn_only=True)
+    print('Wait till packages update ... ')
+    print('YUM UPDATE started at: {0}'.format(time.ctime()))
+    update_packages(quiet=False)
+    print('YUM UPDATE finished at: {0}'.format(time.ctime()))
+    if from_version == '6.0':
+        run('yum install -y capsule-installer', warn_only=True)
+        # Copy answer file from katello to capule installer
+        run('cp /etc/katello-installer/answers.capsule-installer.yaml.rpmsave '
+            '/etc/capsule-installer/answers.capsule-installer.yaml',
+            warn_only=True)
+    # Generates Capsule Certs file on satelltie and copies in capsule
+    execute(
+        generate_capsule_certs,
+        cap_host,
+        True,
+        host=sat_host
+    )
+    # Copying the capsule cert to capsule
+    execute(lambda: run("scp -o 'StrictHostKeyChecking no' {0}-certs.tar "
+                        "root@{0}:/home/".format(cap_host)), host=sat_host)
+    # Rebooting the system again to see possible errors
+    if os.environ.get('RHEV_CAPSULE'):
+        reboot(120)
+    # Stopping the services again which started in reboot
+    run('for i in qpidd pulp_workers pulp_celerybeat '
+        'pulp_resource_manager httpd; do service $i stop; done')
+    # Upgrading Katello installer
+    print('CAPSULE UPGRADE started at: {0}'.format(time.ctime()))
+    if to_version == '6.1':
+        run('capsule-installer --upgrade --certs-tar '
+            '/home/{0}-certs.tar'.format(cap_host))
+    else:
+        run('foreman-installer --scenario capsule --upgrade --certs-tar '
+            '/home/{0}-certs.tar'.format(cap_host))
+    print('CAPSULE UPGRADE finished at: {0}'.format(time.ctime()))
+    # Test The status of all katello services
+    run('katello-service status', warn_only=True)
+
+
+def product_upgrade(
+        product, sat_image=None, cap_image=None):
+    """Task which upgrades the product.
+
+    Product is satellite or capsule.
+
+    :param product: A string. product name wanted to upgrade.
+    :param sat_instance: A string. Openstack Satellite Instance name
```

Comment:
The parameter is not required for this function. So removing the docstring about it.